ecc/bls12-377/fr/mimc/mimc.go

```diff
@@ -98,23 +98,22 @@ func (d *digest) BlockSize() int {
 // If len(p) is not a multiple of BlockSize and any of the []byte in p represent an integer
 // larger than fr.Modulus, this function returns an error.
 //
```

Comment:
add in godoc a warning to say that this method may error (whereas Hash interface{} say it may not)